math/src/lib.rs

```diff
@@ -1,10 +1,17 @@
+#![cfg_attr(feature = "no_std", no_std)]
+
 pub mod cyclic_group;
-pub mod elliptic_curve;
 pub mod errors;
-pub mod fft;
 pub mod field;
 pub mod helpers;
-pub mod msm;
-pub mod polynomial;
 pub mod traits;
 pub mod unsigned_integer;
+// These modules don't work in no-std mode
+#[cfg(not(feature = "no_std"))]
+pub mod elliptic_curve;
+#[cfg(not(feature = "no_std"))]
+pub mod fft;
+#[cfg(not(feature = "no_std"))]
+pub mod msm;
+#[cfg(not(feature = "no_std"))]
+pub mod polynomial;
```

Comment:
_Note for future work_
If we want to support wasm in the future, we could use a structure similar to [cairo-rs](https://github.com/lambdaclass/cairo-rs/blob/d33220728306855ef2c97bc115b73ebc8ae6c45a/vm/src/lib.rs#L13-L16), with an _stdlib_ module, that uses std or no-std alternatives according to the `no_std` feature.

Reply:
Done